instrumentation/redisson-3.0/redisson-3.0.gradle

```
@@ -0,0 +1,23 @@
+// Set properties before any plugins get loaded
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+}
+
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.redisson"
+    module = "redisson"
+    versions = "[3.0.0,)"
+    assertInverse = true
+  }
+}
+
+dependencies {
+  compileOnly group: 'org.redisson', name: 'redisson', version: '3.0.0'
+
+  testLibrary group: 'org.redisson', name: 'redisson', version: '3.0.0'
```

i think this can be consolidated
```suggestion
  library group: 'org.redisson', name: 'redisson', version: '3.0.0'
```

---

Thanks. fixed at lastest commit.